source/algorithms.tex

```diff
@@ -13530,7 +13530,7 @@
 make use of the following exposition-only function templates:
 \begin{codeblock}
 template<class T>
-  constexpr void* @\placeholdernc{voidify}@(T& obj) noexcept {
```

Comment:
could you retain the `nc`-ness (i.e. use `\exposidnc` where appropriate)?

Reply:
I don't think it's appropriate anywhere. Just below this change, we have `deref-move` without `nc`.

It doesn't seem like `voidify` is ever being vertically aligned with something else, and to my understanding, that's when we would disable italics corrections.